sample_data/synonyms.rules.txt

```diff
@@ -41,13 +41,70 @@ they will |they'll
 they'll   |they will
 
 
-okay    | ok
-ok      | okay
+okay    | ok ; 'kay
+ok      | okay ; 'kay
+'kay    | okay ; ok
 
-i'd     |i had ; i would
+til     | until ; 'til
+'til    | until ; til
+until   | 'til ; til
+
+awhile | a while
+a while | awhile
+
+lotta    | a lot of
+a lot of | lotta
+sorta    | sort of
+sort of  | sorta
+
+dunno | don't know ; do not know
+don't know | dunno
+do not know | dunno
+
+lemme | let me
+let me | lemme
+
+lets | let us
+let us | lets
+
+long term | longterm ; long-term
+longterm | long term ; long-term
+long-term | long term ; longterm
```

Comment:
Don't really like these, would like to address compound word errors more holistically rather than expanding this with hard examples

Reply:
I agree wholeheartedly -- I do think there's a better approach to handle this with some other logic! I can remove these and write up a ticket to work on this... Would you say this should be handled by something like dataprep to produce alternate spellings?